Intersect.Server/Entities/Combat/DoT.cs

```diff
@@ -96,11 +96,16 @@ public void Tick()
 
             var deadAnimations = new List<KeyValuePair<Guid, sbyte>>();
             var aliveAnimations = new List<KeyValuePair<Guid, sbyte>>();
-            if (SpellBase.HitAnimationId != Guid.Empty)
+            if (SpellBase.OverTimeAnimationId != Guid.Empty)
             {
-                deadAnimations.Add(new KeyValuePair<Guid, sbyte>(SpellBase.HitAnimationId, (sbyte) Directions.Up));
-                aliveAnimations.Add(new KeyValuePair<Guid, sbyte>(SpellBase.HitAnimationId, (sbyte) Directions.Up));
+                deadAnimations.Add(new KeyValuePair<Guid, sbyte>(SpellBase.OverTimeAnimationId, (sbyte)Directions.Up));
```

Comment:
So I just realized that a KVP doesn't need to be created twice (which introduces an extra line to include a bug) so the KVP should be stored in a variable and then reused for both lists, but...

What is the difference between the over-time animation and the hit animation in this case?

If we use an if statement like this, for DoTs I think they won't have a hit animation anymore which doesn't seem intentional?

Reply:
Hm, sorry if I'm not following - the DoT has either the spell's set DoT animation or, if not that, uses the default hit animation of the spell. If neither, then no animation is added.

The difference between the two is just convenience - otherwise, if we _force_ the user to select an OT animation (and don't assume the default spell one if none selected), the migration will have to be slightly more involved.

Reply:
Let me rephrase, does the on-hit animation still display when the spell hits?

Reply:
Sorry - yes. When the spell first hits, the on-hit animation will play. Subsequent "over time" animations will use whatever is set.